styles.css

```diff
@@ -0,0 +1,970 @@
+  }
+
+  .summit-intro img {
+    width: 20%;
+    position: absolute;
+    top: 14rem;
+    right: 6%;
+    z-index: -1;
+  }
+
+  .summit-intro p {
+    line-height: 2rem;
+  }
+
+  .plz-contact {
+    text-align: center;
+    margin: 2rem 1rem;
+    width: 90%;
+    line-height: 1.7rem;
+  }
+
+  .dt-container {
+    display: flex;
+    flex-direction: row;
+    font-size: 30px;
+    text-align: center;
+    flex-wrap: wrap;
+    margin-top: 5%;
+    background-color: cornsilk;
+    border: 1px solid white;
+    border-radius: 8px;
+    fill-opacity: 0.5;
+    width: 90%;
+  }
+
+  .dt-container h4 {
+    font-weight: 700;
+    color: #272a31;
+    font-size: 1.3rem;
+  }
+
+  .extra-div {
+    display: none;
+  }
+
+  .dt-item1 {
+    flex: 10%;
+    height: 60%;
+  }
+
+  .dt-item1 img {
+    width: 80%;
+  }
+
+  .dt-item2 {
+    flex: 70%;
+  }
+
+  .dt-item3 {
+    flex: 100%;
+    align-content: center;
+  }
+
+  .dt-item3 p {
+    font-size: 1rem;
+    font-weight: 600;
+  }
+
+  .main-program {
+    background-image: url("./images/dot-bg-1.jpg");
+    background-size: 100%;
+    text-align: center;
+  }
+
+  .main-program hr {
+    border-top: 0.3rem solid #ff4f38;
+    margin-top: -0.5rem;
+    width: 15%;
+    margin-bottom: 3rem;
+  }
+
+  .main-program h2 {
+    color: #fff;
+    padding-top: 5%;
+    padding-bottom: 5%;
+  }
+
+  .program-container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+  }
+
+  .program-item {
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+    background-color: #272a31;
+    width: 90%;
+    margin-bottom: 2%;
+    padding: 3%;
+    margin-left: 1rem;
+    margin-right: 1.5rem;
+  }
+
+  .program-img {
+    width: 10%;
+  }
+
+  .program-name {
+    width: 25%;
+    color: #ec5242;
+    font-weight: 700;
+    font-size: 0.9rem;
+    text-align: center;
+    text-overflow: initial;
+    padding-top: 5%;
+  }
+
+  .program-desc {
+    width: 65%;
+    color: white;
+    font-size: 0.8rem;
+    line-height: 1.2rem;
+    align-self: center;
+    padding-left: 3%;
+  }
+
+  .main-program .see-whole {
+    display: none;
+  }
+
+  .join-summit {
+    margin-top: 2rem;
+    padding: 2rem 3rem;
+    background-color: #ff4f38;
+    text-decoration: none;
+    color: white;
+    display: inline-block;
+  }
+
+  .speakers {
+    text-align: center;
+  }
+
+  .speakers h2 {
+    font-weight: 500;
+    font-size: 1.5rem;
+    margin-top: 2rem;
+  }
+
+  .speakers-header-hr {
+    border-top: 0.3rem solid #ff4f38;
+    width: 15%;
+    margin-bottom: 3rem;
+  }
+
+  .speakers-detail {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+  }
+
+  .speaker {
+    flex: 100%;
+  }
+
+  .speaker-flex {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+  }
+
+  .speaker-det {
+    display: grid;
+    grid-template-columns: 25% 75%;
+    align-items: stretch;
+    gap: 10px;
+    padding: 10px;
+    border: 2px solid grey;
```

Comment:
- Kindly make sure to remove the border around your speaker in order to stick to Cindy's design :recycle: 
![Screenshot from 2023-03-16 07-34-17](https://user-images.githubusercontent.com/97834160/225524471-54f9a681-c28f-4b51-8ad5-1fe7e73a6fb9.png)

Reply:
Modified the code and removed the border as per instructions.